opensrp-chw/src/kk/resources/anc_member_registration_sw.properties

```diff
@@ -1,5 +1,5 @@
-anc_member_registration.step1.intervention_consent.label = Fomu ya ridhaa ya usajili: Soma na uelezee  fomu ya ridhaa ya usajili na ukusanyaji wa taarifa kwa mama mjamzito, na umuombe ridhaa ya kushiriki, yeye na familia yake. \
-  Je, kaya imekubali kusajiliwa na taarifa kuchukuliwa/kushirikishwa na wahusika walioidhinishwa?
+anc_member_registration.step1.intervention_consent.label = Fomu ya ridhaa kwa ajili ya Afua ya Malezi, Makuzi Na Maendeleo ya Awali ya Mtoto: Soma na uelezee  fomu ya ridhaa kwa ajili ya Afua ya Malezi, Makuzi, na Maendeleo Ya Awali ya Mtoto kwa mama mjamzito, na umuombe ridhaa kwa ajili yake na kaya yake kushiriki.\
```

Comment:
@maulidkategile  Is it Afua ya malezi or Afya ya malezi? Can you confirm this with Grace

Reply:
@issyzac It is "Afua"